app/views/static/terms_of_use.html.erb

```diff
@@ -41,7 +41,7 @@
       <div class='static-page__row'>
         <a name='terms'></a>
         <p class='static-page__text static-page__text--sinai'><strong><span class='si-link'>*</span></strong> The terms and conditions on this page rehearse the user agreement from the <%= link_to 'Sinai-ID Registration', 'https://sinai-id.org/users/sign_up', target: '_blank', rel: 'noopener', class: 'si-link' %>. Violation of these terms and conditions may result in the suspension of your user account and notification to the Monastery of the violation.</p>
-        <p class='static-page__text static-page__text--sinai'>Users are invited to review the <%= link_to 'Privacy Policy', 'https://sinai-id.org/static_pages/spp_privacy', target: '_blank', rel: 'noopener', class: 'si-link' %> associated with their login. Registration data is managed by the Early Manuscripts Electronic Library (EMEL). If you ever wish to delete your user account, email a request to <%= mail_to 'sinai.id.emel@gmail.com', 'sinai.id.emel@gmail.com', class: 'si-link' %>, and EMEL will remove your account data from its database.</p>
+        <p class='static-page__text static-page__text--sinai'>Users are invited to review the <%= link_to 'Privacy Policy', 'https://sinai-id.org/static_pages/spp_privacy', target: '_blank', rel: 'noopener', class: 'si-link' %> associated with their login. Registration data is managed by the Early Manuscripts Electronic Library (EMEL). If you ever wish to delete your user account, email a request to <%= mail_to 'sinai@library.ucla.edu', 'sinai@library.ucla.edu', class: ' si-link' %>, and EMEL will remove your account data from its database.</p>
```

Comment:
The ticket does not say remove sinai emel id email address

Reply:
confirming that we want to leave the sinai.id.emel@ emails in place :) thanks for catching that!

Reply:
fixed